core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -4612,8 +4612,11 @@ private void setColumnSize(int columnIndex, long size, boolean doubleAllocate) {
                     if (doubleAllocate) {
                         mem2.allocate(pos * Long.BYTES + Long.BYTES);
                     }
-                    mem2.jumpTo(pos * Long.BYTES + Long.BYTES);
-                    m1pos = Unsafe.getUnsafe().getLong(mem2.getAppendAddress() - 8);
+                    // Jump to the number of records written to read length of var column correctly
+                    mem2.jumpTo(pos * Long.BYTES);
```

Comment:
the intent behind is to ensure side, "under-jumping" could mean there is no 8 bytes to read after that. The only reason it works is that page size uniquely divisibly by 8 and this doesn't happen or very hard to make happen. Nonetheless logically this is incorrect